operations/sthree.go

```diff
@@ -375,6 +395,26 @@ func s3syncFlags(args ...cli.Flag) []cli.Flag {
 			Name:  "timeout",
 			Usage: "specify a timeout for operations, defaults to unlimited timeout if not specified",
 		},
+		cli.BoolFlag{
```

Comment:
you can do BoolTFlag, also you can have workers default to 0 and then turn on parallel if you specify a non-zero value for workers

Reply:
Workers defaults to 0 now